### PR TITLE
Remove PhotoPicker from React Native docs

### PR DESCRIPTION
**Pull request by amhinson**
*Issue #, if available:*
https://github.com/aws-amplify/amplify-js/issues/5424#issuecomment-626244344

*Description of changes:*
The docs were inadvertently showing the React `PhotoPicker` documentation for React Native as well, even though there is no `PhotoPicker` implemented for `aws-amplify-react-native` yet, leading to some customer confusion.

This PR just removed any references to `PhotoPicker` for React Native.

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/ui/storage/menu.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "title": "Storage",
     "items": [
-        "photo-picker",
```
Comment (sammartinez) [inline docs/ui/storage/menu.json:4]: Oops, didnt see this, you actually dont need to remove this.

Comment (amhinson) [reply docs/ui/storage/menu.json:4]: Seeing an error when I add this back:
```
UnhandledPromiseRejectionWarning: Error: Ran into problems resolving "/docs/docs/ui/storage/photo-picker.md", referenced from "/docs/docs/ui/storage/menu.json"
```

There are no other framework fragments aside from React Native in `ui/storage`, so I think this should be safe to remove.

Comment (sammartinez) [reply docs/ui/storage/menu.json:4]: Lets talk a look after stand up on this.

Comment (sammartinez) [reply docs/ui/storage/menu.json:4]: as per our offline conversation, this is approved. Re-approving

**Pull request merged**